recipes-automotive/sdl-generic-hmi/sdl-generic-hmi_git.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "HTML based reference HMI for SmartDeviceLink"
+DESCRIPTION = "SmartDeviceLink (SDL) is a standard set of protocols and \
+messages that connect applications on a smartphone to a vehicle head \
+unit."
+HOMEPAGE = "https://www.smartdevicelink.com"
+SECTION = "app"
+
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=cf455e8d39d2ded1d85b1c5dea6ec3c5"
+
+SRC_URI = "git://github.com/smartdevicelink/generic_hmi.git;branch=develop"
+
+PV = "0.0.0+git${SRCPV}"
+SRCREV = "96c44b8f488977fe1cf29af1a20898187a982b63"
+
+S = "${WORKDIR}/git"
+
+PACKAGES = " \
+    ${PN} \
+"
+
+FILES_${PN}_append = " \
+    ${datadir}/smartdevicelink/generic_hmi/ \
+"
+
+do_install() {
+    install -d ${D}${datadir}/smartdevicelink/generic_hmi/build
+    install -m 0644 ${S}/index.html ${D}${datadir}/smartdevicelink/generic_hmi
+    install -m 0644 ${S}/build/bundle.js ${D}${datadir}/smartdevicelink/generic_hmi/build
```

Comment:
@gunnarx  Thanks for the PR. I tested on PC and use Inspect tool on google chrome browser and check the console. It reports some fonts are missing.

> cc56dd09ba0c52a6824aa936024ac810.woff2 Failed to load resource: net::ERR_FILE_NOT_FOUND
> 4e8caa33f4fef80561b267d0b8cd2126.woff2 Failed to load resource: net::ERR_FILE_NOT_FOUND
> 9e06a516bafae247b569ad0c32cd3a57.woff2 Failed to load resource: net::ERR_FILE_NOT_FOUND
> d9f44a669b0ce892cee443367d04adec.woff Failed to load resource: net::ERR_FILE_NOT_FOUND
> d612704147ab38b14c13feaa994a2220.woff Failed to load resource: net::ERR_FILE_NOT_FOUND
> 9b81f8c40379b3fe831234c9895f55d7.woff Failed to load resource: net::ERR_FILE_NOT_FOUND

Please install the fonts folder to ${D}${datadir}/smartdevicelink/generic_hmi/ 

` install -d ${D}${datadir}/smartdevicelink/generic_hmi/fonts`
`install -m 0644 ${S}/fonts/*  ${D}${datadir}/smartdevicelink/generic_hmi/fonts`